requirements.txt

```diff
@@ -5,5 +5,5 @@ elasticsearch>=7.16,<7.17
 PyYAML>=6.0,<6.1
 aws_lambda_typing>=2.16.1,<3
 ujson>=5.5.0,<5.6
-requests>=2.25.1
-
+requests<2.26,>=2.20.0
```

Comment:
particular reasons to bump down min version from `>=2.25.1` to  `>=2.20.0`?

I had a strategy in mind when pinning versions in `requirements.txt` that was to have the latest patch available (helped by the dependantbot) and lock to the next minor.

it's the best I could find to keep up without stale versions and still be quite confident that every builds use more or less the same version without explicitly pinning it.
maybe having `==X.Y.Z` would be as good or more, if we rely on the dependantbot

Reply:
That came from localstack requirements

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
localstack 1.2.0 requires requests<2.26,>=2.20.0, but you have requests 2.26.0 which is incompatible.
localstack-ext 1.2.0 requires requests<2.26,>=2.20.0, but you have requests 2.26.0 which is incompatible.

```

but you are right, makes more sense to just have `==2.25.1`, which is the latest version before 2.26

Reply:
Changed in https://github.com/elastic/elastic-serverless-forwarder/pull/183/commits/7344129a6cbf5007e56d65645d2e863e258fcc2b